components/rich_text_editor/rich_text_editor.vue

```diff
@@ -102,6 +103,14 @@ export default {
       type: [Boolean, Object],
       default: false,
     },
+
+    /**
+     * Placeholder text
+     */
+    placeholderText: {
```

Comment:
I would call this just `placeholder` to align with the native input's attribute.